suite-common/wallet-core/src/transactions/transactionsThunks.ts

```diff
@@ -172,23 +173,25 @@ export const exportTransactionsThunk = createThunk(
 );
 
 export const fetchTransactionsThunk = createThunk(
-    `${modulePrefix}/addFakePendingTransaction`,
+    `${modulePrefix}/fetchTransactionsThunk`,
     async (
         {
-            account,
+            accountKey,
             page,
             perPage,
             noLoading = false,
             recursive = false,
         }: {
-            account: Account;
+            accountKey: string;
```

Comment:
We have type `AccountKey` I think

Reply:
[df99815](https://github.com/trezor/trezor-suite/pull/6319/commits/df99815d2c384b0067e0574d83b832ed98253fc7)